tests/envs/test_pybullet_utils.py

```diff
@@ -18,7 +18,7 @@ def _setup_pybullet_test_scene():
 
     Initialized only once and cached globally for efficiency.
     """
-    global _PYBULLET_SCENE_IS_SETUP  # pylint:disable=global-statement
+    global _PYBULLET_SCENE_IS_SETUP  # pylint: disable=global-statement
```

Comment:
@tomsilver can we use the fixture here too :)

Reply:
Yup will do tomorrow

Reply:
#554